cn/examples/section_3/functions.md

```diff
@@ -1,6 +1,6 @@
 ---
 version: 1.0.0
-example_title: Functions
+example_title: 函数

```

Comment:
Hey! Thanks for contributing but can you close your other 2 PRs and make all the changes in this PR only?